css/states.css

```diff
@@ -2,3 +2,36 @@
   width: device-width;
   zoom: 1;
 }
+
+@media all and (min-width: 640px) {
```

Comment:
Many of these styles should be in modules or layout. The states.css file is only for changes in the state of the page, such as the `.menu-close-button-container` styles. The `@media` query is not a change in the state.